its/projects/DefaultRuleParametersCanBeCustomized/DefaultRuleParametersCanBeCustomized/DefaultRuleParametersCanBeCustomized.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Credentials
+{
+    public class Credentials
+    {
+        private const string Username = "admin";
+        private const string Password = "Password123";
+        private const string Usario = "admin";
```

Comment:
the test is a bit misleading - I expect 'Usario' is 'username' and an issue is raised on it because 'usario' is in given as a name for credential, together with "password, pwd, senha" (senha=password in portuguese, apparently).

The problem is that an issue is raised on line 7 and not on line 5 and this leads to some cognitive overload when reading the test - why is the rule raising for portuguese username but not for english username?

So I would keep just 'senha' and keep the test easy to follow (avoid raising on 'username' and 'usario')